_data/sw_clients.yml

```diff
@@ -149,6 +174,8 @@
           sasl: 19+
         v3.2:
           monitor: 15+
+        SASL:
+          - plain
```

Comment:
mIRC supports SASL? I thought only by a whacky "third-party" script. There's no GUI for it as far as I know, and the third party script was something I personally never could get working (hence me piping my connection through ZNC)